spec/renuo/bin-check/servant_thread_spec.rb

```diff
@@ -81,4 +81,17 @@
       expect(servant.run).to have_attributes(result_attributes)
     end
   end
+
+  context 'appended output and error_output' do
+    let(:script) { build :with_appended_output_script }
+    let(:servant) { RenuoBinCheck::ServantThread.new(script) }
+
+    let(:result_attributes) { attributes_for :appended_output_result }
+
+    after(:each) { FileUtils.remove_dir('./tmp/bin-check') }
+
+    it 'uses overridden output' do
+      expect(servant.run).to have_attributes(result_attributes)
+    end
+  end
```

Comment:
Would it make sense to dry up the tests? Maybe this goes to far, but you could also have a look at how tests should be written by third party contributors. Is it for example possible to simulate the behavior of a command?